proptest/src/test_runner/runner.rs

```diff
@@ -210,6 +212,96 @@ where
     })
 }
 
+#[cfg(feature = "handle-panics")]
+mod panicky {
+    use std::boxed::Box;
+    use std::cell::Cell;
+    use std::panic::{set_hook, take_hook, PanicInfo};
+    use std::sync::Once;
+    use std::{mem, ptr};
+
+    thread_local! {
+        // Pointers to arbitrary fn's are fat, and Rust doesn't allow crafting null pointers
+        // to fat objects. So we just store const pointer to tuple with whatever data we need
+        static HANDLER: Cell<*const (*mut dyn FnMut(&PanicInfo<'_>),)> = Cell::new(ptr::null());
+    }
+
+    static INIT_ONCE: Once = Once::new();
+    // NB: no need for external sync, value is mutated only once, when init is performed
+    static mut DEF_HOOK: Option<Box<dyn Fn(&PanicInfo<'_>) + Send + Sync>> =
```

Comment:
please expand this to `DEFAULT_HOOK` for clarity, i'm concerned that "DEF" could be mistaken for something else

Reply:
Done